cdap-standalone/src/main/java/co/cask/cdap/StandaloneMain.java

```diff
@@ -386,6 +402,7 @@ public static void main(String[] args) {
         System.err.println("Failed to start Standalone CDAP");
         e.printStackTrace(System.err);
       }
+      main.stopExternalProcesses();
```

Comment:
I honestly don’t remember! But there was some case where main was null, and that was causing a problem..and so moving the create method up (though outside of the try block) helped address that...